internal/coreinternal/testdata/profile.go

```diff
@@ -52,7 +52,6 @@ func GenerateProfilesTwoProfilesSameResource() pprofile.Profiles {
 
 func fillProfileOne(profile pprofile.Profile) {
 	profile.SetStartTime(TestProfileStartTimestamp)
-	profile.SetEndTime(TestProfileEndTimestamp)
```

Comment:
is end time no longer needed?

Reply:
The function was removed in https://github.com/open-telemetry/opentelemetry-collector/pull/11796 so hopefully not